example/src/main/java/com/transferzero/test/Application.java

```diff
@@ -7,11 +7,11 @@
 import com.transferzero.sdk.api.SendersApi;
 import com.transferzero.sdk.api.TransactionsApi;
 import com.transferzero.sdk.model.*;
-import org.joda.time.LocalDate;
+import okhttp3.Request;
+import okhttp3.Response;
 
 import java.math.BigDecimal;
-import java.util.ArrayList;
-import java.util.UUID;
+import java.time.LocalDate;
```

Comment:
What's the reason for replacing `joda.time.LocalDate` with `java.time.LocalDate`?